### ISSUE TITLE
[Instance][Shadow Labs] Grandmaster Vorpil encounter - Moving Shadow Voids

### ISSUE THREAD

**Issue by nemmjuuk** [dalaranwow/dalaran-wow#5249]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Encounter with Grandmaster Vorpil on normal mode:
As soon the encounter is startet,  shadow voids/portals appear. The shadow voids move towards the player and start dealing melee damage. 

**Expected behaviour**: Tell us what should happen instead
The shadow voids are suppose to be stationary, non-aggressive and just spawn points for "Void Travelers". 

**Steps to reproduce the problem**:

1. Start the encounter and watch what happens
2. 
3. 

**Include proofs for this behaviour**
Combat log entries: 
"Void Portal's melee swing hits <playername> for 876 Physical."

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided
http://wowwiki.wikia.com/wiki/Grandmaster_Vorpil
https://www.youtube.com/watch?v=1zn538fRQ98

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Note: The void portals melee damage by far exceeds the damage cause by the boss itself and I found no way to survive the encounter (lvl 80 warlock). In the current state shadow labs cannot be completed.

**Comment by TeridanDW** [comment]
Works as intended on PTR.

**Comment by Jindra555** [comment]
Still happening. So much stuff never gets pushed to live.

**Comment by Rushor** [comment]
It only takes time.

**Comment by Sithis288** [comment]
Not resolved.

**Comment by Golaman2** [comment]
Not fixed.

**Comment by taichouVu** [comment]
Still not fixed or it broke again. Just got wiped by moving portals that are untargetable.

**Comment by taichouVu** [comment]
For anyone else stuck on this. 

The work around is to have the tank start combat and fight alone until the portals spawn void walkers and disappears. Healing pots and damage mitigation to stay alive until then. After the void portals disappear, the fight proceeds as normal.
